metadata_mapper/mappers/oai/samvera_mapper.py

```diff
@@ -3,12 +3,70 @@
 
 class SamveraRecord(OaiRecord):
 
+    TYPES = {
+        "img": "image",
+        "txt": "text"
+    }
+
+    TYPES_BASEURL = "http://id.loc.gov/vocabulary/resourceTypes/"
+
     def UCLDC_map(self):
         return {
-            'is_shown_at': self.source_metadata.get('isShownAt'),
-            'is_shown_by': self.source_metadata.get('isShownBy')
+            "alternativeTitle": self.map_alternative_title,
+            "type": self.map_type,
         }
 
+    def map_alternative_title(self) -> list:
```

Comment:
I think this function should be moved to the OAI mapper.

Reply:
Will do.